### PR TITLE
Adding new flow field for benchmarking

### PR DESCRIPTION
**Pull request by GuillermoCasas**
Adding new object field to benchmark derivative recovery.

### CODE REVIEW

**Comment by pooyan-dadvand** [comment]
@GuillermoCasas and @roigcarlo would you please check and fix the invalid-email-address as the committer?

File: .gitignore [hunk 1]
```diff
@@ -50,3 +50,9 @@ kratos/includes/kratos_version.h
 # Output of tests of meshing application (MMG)
 *.o.sol
 *.o.mesh
+
+cmake_build/
```
Comment (loumalouomega) [inline .gitignore]: You should add your personal excludes to the .git/info/exclude file in your Kratos installation, the files in the .gitignore should be common files to everyone

https://github.com/KratosMultiphysics/Kratos/wiki/How-to-make-Git-ignore-the-files-resulting-from-a-compilation-without-conflicts-in-.gitignore

**Review by loumalouomega — CHANGES_REQUESTED** [review]
Please keep the .gitignore of the master branch and add your custom ignores to .git/info/exclude

**Comment by roigcarlo** [comment]
@pooyan-dadvand I would need @GuillermoCasas to pm a public git mail so I can associate it, or alternatively, he can add his cimne mail to his account, verify it and it will be corrected automatically

File: .gitignore (hunk 1 above)
Comment (roigcarlo) [reply .gitignore]: I would like to add that cmake_build is already ignored anyway: https://github.com/KratosMultiphysics/Kratos/blob/62e58da2bf9b9a69ab4c64bf307fa37369bb92e2/.gitignore#L5

**Review by maceligueta — CHANGES_REQUESTED** [review]
Definitely all these files resulting from the compilation should not be added to the repo.

**Review by loumalouomega — CHANGES_REQUESTED** [review]
Remove everything related with cmake_build_debug

**Comment by maceligueta** [comment]
@GuillermoCasas : remember that many of us followed the procedure explained in the wiki: http://kratos-wiki.cimne.upc.edu/index.php/How_to_make_Git_ignore_the_files_resulting_from_a_compilation_without_conflicts_in_.gitignore

**Comment by GuillermoCasas** [comment]
Thank you, guys. Yes, I was struggling with these operations. I think I finally managed. Slowly getting the hang of it.

**Comment by maceligueta** [comment]
Ok, try to remove those 6 configuration files (out of 8 total) from the commit.

**Review by loumalouomega — COMMENTED** [review]
Remove the files generated by qtcreator

**Comment by loumalouomega** [comment]
@maceligueta Please use the new wiki https://github.com/KratosMultiphysics/Kratos/wiki/How-to-make-Git-ignore-the-files-resulting-from-a-compilation-without-conflicts-in-.gitignore

**Review by GuillermoCasas — COMMENTED** [review]
Basically accepting changes made by Vicente and adding what is needed for the application to compile.

**Review by loumalouomega — APPROVED** [review]

**Comment by loumalouomega** [comment]
@maceligueta I think now is ready to merge

**Review by maceligueta — APPROVED** [review]

**Pull request merged**